src/dataExplorer/components/SaveAsCellForm.tsx

```diff
@@ -48,9 +53,9 @@ interface State {
 interface OwnProps {
   dismiss: () => void
 }
-
+type RouterProps = RouteComponentProps<{orgID: string}>
```

Comment:
use the `getOrg` selector to get the org id. views shouldn't really be coupled with the route